docs/app-rails/technical-foundation.md

```diff
@@ -129,7 +129,9 @@ Rails has some useful [built-in debugging tools](https://guides.rubyonrails.org/
 
```

Comment:
Instead of:

```md
Rails has some useful [built-in debugging tools](https://guides.rubyonrails.org/debugging_rails_applications.html).
```

Replace with:

```md
Rails has some useful [built-in debugging tools](https://guides.rubyonrails.org/debugging_rails_applications.html). Here are a few different options:
```